stepengine/src/main/java/org/tweetwallfx/devoxx18be/steps/mosaic/ImageMosaicCreateStep.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2016-2018 TweetWallFX
```

Comment:
new class should only be 2018 in years
```suggestion
 * Copyright 2018 TweetWallFX
```